notebooks/Integrate_NLP_Libraries.ipynb

```diff
@@ -174,7 +174,7 @@
     {
```

Comment:
<p>This adds our new mentions DataFrame</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>